app/models/series.rb

```diff
@@ -104,4 +108,23 @@ def schedule_for_episode_number(episode_number)
 
     schedule_index
   end
+
+  def destroy
+    if v4?
+      really_destroy!
```

Comment:
(b/c I don't have a problem with empty v4 series showing up in v3, this isn't like pieces/stories where it will break things)